npc/re/other/dimensional_gap.txt

```diff
@@ -0,0 +1,549 @@
+	next();
+	mes("[Expedition Guide]");
+	mes("Anyone who has become lost here has yet to return.");
+	close();
+}
+
+mid_camp,212,290,4	script	Expedition Guide#02	4_M_EIN_SOLDIER,{
+	mes("[Expedition Guide]");
+	mes("Please use a ^0000FFTransport Device^000000 if you wish to travel through the dimensional rift. The insides have not been fully explored yet so please be careful in there.");
+	close();
+}
+
+moc_fild22b,225,198,6	duplicate(Expedition Guide#01)	Expedition Guide#03	4_M_MOC_SOLDIER
+moc_fild22b,223,203,4	duplicate(Expedition Guide#02)	Expedition Guide#04	4_M_MOC_SOLDIER
+bif_fild01,314,155,6	duplicate(Expedition Guide#01)	Expedition Guide#05	4_M_EIN_SOLDIER
+bif_fild01,320,154,4	duplicate(Expedition Guide#02)	Expedition Guide#06	4_M_EIN_SOLDIER
+dic_fild02,236,33,6	duplicate(Expedition Guide#01)	Expedition Guide#07	4_M_EIN_SOLDIER
+dic_fild02,239,36,4	duplicate(Expedition Guide#02)	Expedition Guide#08	4_M_EIN_SOLDIER
+
+dali,137,87,4	script	Allied Forces Soldier	4_M_MOCASS1,{
+	mes("[Allied Forces Soldier]");
+	mes("This dimensional rift is connected to the Allied Forces ^0000FFExpedition Camp^000000 in Jotunheim.");
+	next();
+	mes("[Allied Forces Soldier]");
+	mes("Do not worry, it has been set up after testing hundreds of experiments for safe teleportation.");
+	close();
+}
+
+dali,122,60,4	script	Morroc Soldier	4_M_MOC_SOLDIER,{
+	mes("[Morroc Soldier]");
+	mes("This dimensional rift is connected to ^0000FFMorroc^000000 back in Rune Midgard.");
+	next();
+	mes("[Morroc Soldier]");
+	mes("This is the fastest gateway back to our world. The area is still destroyed but restoration efforts will soon be on the way.");
+	close();
+}
+
+dali,51,104,6	script	Spledide Soldier	4_M_FAIRYSOLDIER2,{
+	mes("[Spledide Soldier]");
+	mes("This dimensional rift is connected to the south of ^0000FFBifrost^000000. It enables quick travel to the Splendide frontline.");
+	next();
+	mes("[Spledide Soldier]");
+	mes("If we use it properly, it enables us to advance more easily to the capital of the Sapha.");
+	next();
+	mes("[Spledide Soldier]");
+	mes("The Mayor has asked us to cooperate as best we can instead of fighting, so I'm under orders not to do anything. Since we have a common enemy we won't fight the Sapha... for now...");
+	close();
+}
+
+dali,56,126,4	script	Dispatched Sapha	4_MAN_BENKUNI,{
+	mes("[Dispatched Sapha]");
+	mes("This dimensional rift is connected to the southern area of the ^0000FFKamidal Mountains^000000 in El Dicastes. I just dropped by for my research, but it is incredible.");
+	next();
+	mes("[Dispatched Sapha]");
+	mes("Moreover it is next to the Laphine capital..... Hmm... Do not worry, I do not want to fight in such a place.");
+	close();
+}
+
+dali,109,94,4	trader	Sundries Merchant	4_M_MERCAT2,{
+OnInit:
+	sellitem Spectacles;
+	sellitem Arrow;
+	sellitem Booby_Trap;
```

Comment:
missing end;
(It would however end with this too, but having end; would be nice)